runtime/lua/vscode.lua

```diff
@@ -1,27 +1,27 @@
 require("vscode.defaults")
 require("vscode.cursor")
```

Comment:
generally a red flag if require() has side-effects. Modules should only return a table, so consumer modules use them like so:

local foo = require('vscode.cursor')